tests/integration/profile.test.ts

```diff
@@ -36,32 +41,55 @@ describe("profile", () => {
     return profile
   }
 
-  it("Sets the display name and role for new users", async () => {
+  async function testDBConnection() {
+    const newUser = fakeUser()
+    await setNewProfile(newUser)
+    let profile = await getProfile(newUser)
+    return profile
+  }
+
+  it("tests db conn", async () => {
+    const profile = await testDBConnection()
+    expect(profile).toBeDefined()
+  })
+
```

Comment:
Not quite sure what's actually being tested here ? Mind if you provide a little context?